.github/workflow-templates/test-dapp.yml

```diff
@@ -52,6 +52,11 @@ jobs:
           console.log(branch);
           return branch;
 
+    - name: install node-canvas dependencies
+      run: |
+        set -e
+        sudo apt update -y
+        sudo apt install -y libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev
```

Comment:
Making the same fix in 11 places... whee! Programming in yaml is ... oh well.

Reply:
> p.s. Why do we need cairo again? A charting thingy in SwingSet IIRC. Maybe that should go somewhere other than agoric-sdk? (IOU a separate issue on that one, I suppose.)

Please do! That one charting thing has caused endless pain because of the extraneous things it pulls in.